spec/acceptance/ini_subsetting_spec.rb

```diff
@@ -1,19 +1,19 @@
 require 'spec_helper_acceptance'
 
-tmpdir = default.tmpdir('tmp')
-
 describe 'ini_subsetting resource' do
+  basedir = setup_test_directory
```

Comment:
can this live in a before all ?

Reply:
I tried that, but between tests basedir is nil